ui/src/common/components/LabelWithHelp.jsx

```diff
@@ -7,8 +7,7 @@ class LabelWithHelp extends Component {
     const { help, label } = this.props;
     return (
       <span>
-        {label}&nbsp;
```

Comment:
why are you changing those?

Reply:
We were using `{' '}` in some places and `&nbsp;` in others. I haven't found if there's a best practice for whitespaces in JSX, but we should stick to one of them.
(In this particular case we don't even need the explicit whitespace since both expressions can fit in the same line)